test_runner/src/main/kotlin/ftl/ios/xctest/common/ParseObjTests.kt

```diff
@@ -0,0 +1,27 @@
+package ftl.ios.xctest.common
+
+import ftl.config.FtlConstants
+import ftl.util.Bash
+
+internal fun parseObjcTests(binary: String): List<String> {
+    installBinaries
+    validateIsFile(binary)
+
+    val results = mutableListOf<String>()
+    // https://github.com/linkedin/bluepill/blob/37e7efa42472222b81adaa0e88f2bd82aa289b44/Source/Shared/BPXCTestFile.m#L18
+    // must quote binary path in case there are spaces
+    var cmd = "nm -U ${binary.quote()}"
+    if (!FtlConstants.isMacOS) cmd = "PATH=~/.flank $cmd"
```

Comment:
has this been tested on windows?

Reply:
nope

Reply:
its gonna break it i see it 🗡️

Reply:
```parseObjcTests``` is used only on ios tests. Flank not supporting ios tests on windows so I think it's fine.